packages/@glimmer-workspace/build/lib/config.js

```diff
@@ -315,17 +316,25 @@ export class Package {
         commonjs(),
         nodeResolve(),
         ...this.replacements(env),
-        ...(env === 'prod' ? [terser({
-          compress: {
-            ecma: 2020,
-            hoist_funs: true,
-            module: true,
-            passes: 3,
-            toplevel: true,
-            unsafe_undefined: true,
-          },
-          module: true,
-        })] : []),
+        ...(env === 'prod'
+          ? [
+              strip({
```

Comment:
I wonder if we can also have this strip the local-debug-flags functions, too?

That would help out the error recovery branch a fair bit.

Tho, the error recovery local debug flags ale entirely different from main

Reply:
@NullVoxPopuli we could have babel plugin for it, as I wrote for `unwrap`

Reply:
I wonder if maybe we need terser in dev mode as well, but with less compression options, and more for constant value function elimination.. hmmm